packages/prettier-rules/package.json

```diff
@@ -2,12 +2,5 @@
   "name": "@uifabric/prettier-rules",
   "version": "7.0.2",
   "description": "Shared Prettier rules for UI Fabric projects",
-  "main": "prettier.config.js",
-  "scripts": {
-    "clean": "",
-    "build": "just-scripts no-op"
-  },
-  "devDependencies": {
-    "just-scripts": "~0.25.2"
-  }
+  "main": "prettier.config.js"
```

Comment:
If you're deleting all the other stuff from this file shouldn't you also delete the project's just config?

Reply:
Ah yes, might have been a miss because prettier techncially isn't built